credentials.properties.enc

```diff
@@ -1,2 +1 @@
-A�h�/=�5��D�Yk���n�6�os�&�l���ܪ��!�h��է�Ͷ��n;Zc2�
-��
\ No newline at end of file
+�d$l�d`�;��e��Q;w8=?�x*kGw��r�¤Z}�CSS��J�Prpr���-�ML>�
```

Comment:
They are the same as all others, but I don't get why is it showed as text, since `*.enc binary` is declared in `.gitattrubures`.